flashcards/backend/decks/models.py

```diff
@@ -20,3 +44,52 @@ def __str__(self):
     class Meta:
         verbose_name = "category"
         verbose_name_plural = "categories"
+
+
+class Deck(TimeStampModel):
+    """Model for representing deck data
+
+    :param image: deck image
+    :type image: file, optional
+    :param name: deck name
+    :type name: str
+    :param is_public: indicates if deck is public
+    :type is_public: bool, optional
+    :param price: deck price
+    :type price: Decimal, min_value=0.00
+    :param author: deck author
+    :type author: User
+    :param popularity: deck popularity
+    :type popularity: int, optional
+    :param rating: deck rating
+    :type rating: int, optional
+    :param is_active: indicates if deck is active
+    :type is_active: bool, optional
+    :param description: deck description
+    :type description: str
+    """
+
+    image: models.FileField = models.FileField(
```

Comment:
I would pass on those typings,  if you run `docker compose exec backend mypy decks/models.py` without them then there is no errors, this is because we have `django-stubs-ext` package

Reply:
what do you mean here? i dont get it?

Reply:
You don't have to do type hints, our models are typed with [django-stubs-ext](https://pypi.org/project/django-stubs-ext/) so just remove them

Reply:
done